roles/dosomething.varnish/vars/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+varnish_prereqs:
+    - apt-transport-https
```

Comment:
No need for vars file here. It makes more sense to list this directly in task:
- This arrays are not designed mutable
- There's only one occurrence of each array